### DISCUSSION TITLE
Support for UTF-8

### DISCUSSION THREAD

**Discussion by felipekutz** [vgalin/html2image#51] [Ideas]
Would be nice whether you add  <meta charset="UTF-8"> at html2image.py line 370, or some support to enable it. Since words in other idioms like Portuguese will not behave well. For example: Você, também, and any other word with special letter. Currently I'm using a workaround to add this feature. 

```
html="<h1>Você, Também</h1>"

css = """ 
    h1 {
         color: green;
    }
</style>
<meta charset="UTF-8"> 
<style>
"""
hti.screenshot(html_str=html, css_str=css, size=(625,600), save_as="test.jpg")
```

**Comment by vgalin** [comment]
Hello, this probably should be an issue instead of a discussion.
However, I'm unable to reproduce your issue, the accentuated characters seems to be correctly rendered (see attachments)
![test](https://user-images.githubusercontent.com/47885533/141652085-7cf3c4ec-e112-4a42-93f2-57ab09b919f4.jpg)

**Reply by felipekutz** [reply]
> Hello, this probably should be an issue instead of a discussion. However, I'm unable to reproduce your issue, the accentuated characters seems to be correctly rendered (see attachments) ![test](https://user-images.githubusercontent.com/47885533/141652085-7cf3c4ec-e112-4a42-93f2-57ab09b919f4.jpg)

hm.. I don't know what going on, because, when I don't use the workaround that I've mentioned before I got the following result:

![Screenshot from 2021-11-16 15-46-52](https://user-images.githubusercontent.com/12447036/142046998-c2a2082b-4328-4a5f-9d78-e12b95791adf.png)

EDIT: Observe that the workaround is the following snippet at the end of css variable:  

```
</style>
meta charset="UTF-8"
<style>

```
The image above represents the case where I've removed the snippet.
